src/test/java/it/gov/pagopa/arc/utils/TestUtils.java

```diff
@@ -59,4 +62,25 @@ public static KeyPair genKeyPair() throws NoSuchAlgorithmException {
         return keyGen.generateKeyPair();
     }
 
+    public static void assertAllFieldsPopulated(Object dto, List<String> excludedFieldList) {
```

Comment:
Use varargs instead of explicit List: this will make more easly to call this method:
https://github.com/pagopa/idpay-payment/blob/main/src/test/java/it/gov/pagopa/common/utils/TestUtils.java#L45

Moreover, I don't really like this name, something like `assertNotNullFields` sounds better to me